clients/ui/frontend/tsconfig.json

```diff
@@ -37,9 +31,10 @@
     "**/*.tsx",
     "**/*.jsx",
     "**/*.js",
-    ".eslintrc.js"
   ],
   "exclude": [
-    "node_modules"
+    "node_modules",
+    "dist",
+    "src/__tests__/cypress"
```

Comment:
There's a public library generated by cypress too, have you considered that?

Reply:
I know that's what we do in ODH, but when I added the Cypress setup I intentionally removed the separate testing build dir and decided to have our normal build and cypress build use the `dist` directory. See the last two cypress commands here https://github.com/kubeflow/model-registry/blob/main/clients/ui/frontend/package.json#L28

Reply:
We can always change this btw I just thought it was simpler this way. Maybe there's a benefit to having a separate build dir for normal build vs testing build

Reply:
oh right, that's my bad, was checking the odh dashboard commands, good call here!